emoji-selector@maestroschan.fr/metadata.json

```diff
@@ -5,7 +5,8 @@
 		"3.32",
 		"3.34",
 		"3.36",
-		"3.38"
+		"3.38",
+    "40.0"
```

Comment:
Sorry for a nitpick, but it seems like it needs a reformat

Reply:
This is fixed in my master branch and included in this PR.